src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -380,6 +381,34 @@ static int FindPolicyInfo(Eventinfo *lf, char *policy, int *socket) {
     return retval;
 }
 
+static int FindPolicySHA256(Eventinfo *lf, char *policy, int *socket, char *wdb_response) {
+
+    char *msg = NULL;
+    char *response = NULL;
+    int retval = -1;
+
+    os_calloc(OS_MAXSTR, sizeof(char), msg);
+    os_calloc(OS_MAXSTR, sizeof(char), response);
+
+    snprintf(msg, OS_MAXSTR - 1, "agent %s sca query_policy_sha256 %s", lf->agent_id, policy);
+
+    if (pm_send_db(msg, response, socket) == 0) {
+        if (!strcmp(response, "ok not found")){
+            retval = 1;
+        }
+        else if (strstr(response, "err")){
+            retval = -1;
+        } else {
```

Comment:
We are not checking here the result is the expected: `ok found <sha256>`. Why don't we follow the same structure as here? https://github.com/wazuh/wazuh/blob/99cfc0dc01bcd277b2e78b1841e6c2f768ee9616/src/analysisd/decoders/security_configuration_assessment.c#L220